charts/opentelemetry-demo-datadog/otelcol-config.yaml

```diff
@@ -0,0 +1,192 @@
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: "otelcol"
+          scrape_interval: 10s
+          static_configs:
+            - targets: ["0.0.0.0:8888"]
+          metric_relabel_configs:
+            - source_labels: [__name__]
+              regex: ".*grpc_io.*"
+              action: drop
+  k8s_cluster:
+    collection_interval: 10s
+  hostmetrics:
+    collection_interval: 10s
+    scrapers:
+      paging:
+        metrics:
+          system.paging.utilization:
+            enabled: true
+      cpu:
+        metrics:
+          system.cpu.utilization:
+            enabled: true
+      disk:
+      filesystem:
+        metrics:
+          system.filesystem.utilization:
+            enabled: true
+      load:
+      memory:
+      network:
+      processes:
+  otlp:
+    protocols:
+      grpc:
+      http:
+
+  filelog/adservice:
+    include_file_path: true
+    include_file_path_resolved: true
+    include_file_name_resolved: true
+    start_at: end
+    poll_interval: 500ms
+    include:
+      - /var/log/pods/**/*ad-service*/*.log
+    operators:
+      # Detect pod, namespace, and container names from the file name.
+      - id: parse_body
+        type: json_parser
+        parse_to: body
+      - id: parse_log
+        type: json_parser
+        parse_from: body.log
+      - id: trace
+        type: trace_parser
+        trace_id:
+          parse_from: attributes.contextMap.trace_id
+        span_id:
+          parse_from: attributes.contextMap.span_id
```

Comment:
Next step should be getting rid of this. We need to make this as easy for the user as possible. This type of configuration in the documentation does not look very well.

Reply:
yup .. agree .. we can follow the pattern are using for log level & message ..